lib/forklift/box_factory.rb

```diff
@@ -76,7 +84,7 @@ def process_versions(config, versions)
 
     def layer_base_box(box)
       return box unless (base_box = find_base_box(box['box']))
-      base_box.merge(box)
+      base_box.deep_merge(box)
```

Comment:
What verison of Vagrant are you on? Cause this patch has been working fine for me for a few days now.